confdb/aclocal_cc.m4

```diff
@@ -714,7 +716,7 @@ int main( int argc, char *argv[] )
     struct { char a; long b; } char_long;
     struct { char a; int b; char c; } char_int_char;
     struct { char a; short b; char c; } char_short_char;
-#ifdef HAVE_LONG_LONG_INT
+#if SIZEOF_LONG_LONG > 0
```

Comment:
`long long` is C99.  At this point we assume C99 in mpich.  So if `sizeof(long long)` is `0`, we need to abort.

Reply:
The current code does not assume the availability of "long long" or "long double". 

Whether to support "long long" or "long double" is more about using the library rather than the library itself. So even the library need C99 to build, the user may still prefer those types to be invalid -- I am not making a strong argument, just playing a cautious role here.

I am OK to require the availability of these types.

Reply:
I see no reason to retain those checks.  Those are C99, and we require C99 compilers at this point.

Reply:
Ok.